### PR TITLE
MTV-726: Changed RHV version to 4.4 SP1 or later

### PR DESCRIPTION
**Pull request by anarnold97**
[MTV-726](https://issues.redhat.com/browse/MTV-726)

Changed compatibility guide module

image

### CODE REVIEW

**Comment by anarnold97** [comment]
@ahadas && @RichardHoch - please can you review this for me?

Thanks

**Review by ahadas — APPROVED** [review]
it's very unlikely that there will be another version of RHV post 4.4 SP1 so we can theoretically drop the 'or later' part but including it just in case is alright

**Comment by anarnold97** [comment]
> it's very unlikely that there will be another version of RHV post 4.4 SP1 so we can theoretically drop the 'or later' part but including it just in case is alright

@ahadas - better safe than sorry and leave `and later` i think. let us see what Richard says

File: documentation/modules/compatibility-guidelines.adoc [hunk 1]
```diff
@@ -12,5 +12,5 @@ You must install compatible software versions.
 .Compatible software versions
 |===
 |{project-full} |{ocp} |{virt} |VMware vSphere |{rhv-full}|OpenStack
-|2.5.0 |4.12 or later |4.12 or later |6.5 or later |4.4.9 or later|16.1 or later
+|2.5.0 |4.12 or later |4.12 or later |6.5 or later |4.4 SP1 or later|16.1 or later
```
Comment (RichardHoch) [inline documentation/modules/compatibility-guidelines.adoc:15]: 2.5.1. no?

**Comment by RichardHoch** [comment]
> > it's very unlikely that there will be another version of RHV post 4.4 SP1 so we can theoretically drop the 'or later' part but including it just in case is alright
> 
> @ahadas - better safe than sorry and leave `and later` i think. let us see what Richard says

I'd leave "or later," unless you think it's promising something. At this point, just leave it.

**Comment by RichardHoch** [comment]
Just one question (MTV 2.5.1 or 2.5.0); otherwise, LGTM.

File: documentation/modules/compatibility-guidelines.adoc (hunk 1 above)
Comment (anarnold97) [reply documentation/modules/compatibility-guidelines.adoc]: Updated

**Comment by anarnold97** [comment]
Added RHV 4.3 deprecation support:

image

**Comment by anarnold97** [comment]
image

**Pull request merged**